src/wallet/wallet.cpp

```diff
@@ -3148,12 +3151,19 @@ bool CWallet::FundTransaction(CMutableTransaction& tx, CAmount &nFeeRet, int& nC
     
     CCoinControl coinControl;
     coinControl.fAllowOtherInputs = true;
-    BOOST_FOREACH(const CTxIn& txin, tx.vin)
+    for (const CTxIn& txin: tx.vin)
         coinControl.Select(txin.prevout);
 
+    // if we have ceased sc withdrawl inputs, they must be taken into account when doing computations for funding input amount
+    CAmount cswInTotAmount = 0;
```

Comment:
`CAmount cswTotAmount = tx.GetCSWValueIn()` instead of the loop.

Reply:
tx is a mutable obj and does not have such method

Reply:
right.